spec/factories/given_answer.rb

```diff
@@ -5,4 +5,15 @@
     is_correct true
     question
   end
+
+  trait :with_question_given_answers do
```

Comment:
This trait would better fit inside the `question` factory, and could be named something like `with_given_answers`.